### DISCUSSION TITLE
wsl-gvproxy.exe security issues

### DISCUSSION THREAD

**Discussion by gcurrier** [sakai135/wsl-vpnkit#258] [Q&A]
So of course, I want to use this within a corporate network, behind a VPN. However, such .exe files inevitably get flagged by BeyondTrust. Next follows the interrogation of why do I want to use this, what is it for, etc. After I announce the reasoning behind this, I get the "standby" signal while the exe in question is scanned for malicious software.

Well, it turns out there are all kinds of "wrong" with it (whether there actually ARE issues is up for debate). I don't get to argue the point - the person running the scan is a button pusher and block checker. If it doesn't pass, there's no appeal. No unblocking the wsl-gvproxy.exe from the BeyondTrust wall.

Is there another way to use this that won't run afoul of more-or-less standard corporate security measures? Or should I just forget about it?

**Comment by sakai135** [comment]
You will need to get gvproxy.exe unblocked somehow to use wsl-vpnkit. If it helps, wsl-vpnkit uses the unmodified exe from https://github.com/containers which is a well-known org around podman. 

If your org allows Docker Desktop or Podman Desktop, both offer similar methods to wsl-vpnkit for getting around VPN restrictions.
